config/sync/core.entity_view_display.node.driving_instructor.diff.yml

```diff
@@ -0,0 +1,107 @@
+uuid: 7bf53967-7d8d-4189-b611-0f7bbab501ad
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.diff
+    - field.field.node.driving_instructor.field_di_adi_no
+    - field.field.node.driving_instructor.field_di_areas
+    - field.field.node.driving_instructor.field_di_categories
+    - field.field.node.driving_instructor.field_di_firstname
+    - field.field.node.driving_instructor.field_di_lastname
+    - field.field.node.driving_instructor.field_email_address
+    - field.field.node.driving_instructor.field_link_url
+    - field.field.node.driving_instructor.field_meta_tags
+    - field.field.node.driving_instructor.field_telephone
+    - node.type.driving_instructor
+  module:
+    - layout_builder
+    - link
+    - telephone_plus
+    - user
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: false
+id: node.driving_instructor.diff
+targetEntityType: node
+bundle: driving_instructor
+mode: diff
+content:
+  content_moderation_control:
+    weight: 0
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_di_adi_no:
+    type: string
+    weight: 1
+    region: content
+    label: inline
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+  field_di_areas:
+    type: entity_reference_label
+    weight: 8
+    region: content
+    label: inline
+    settings:
+      link: false
+    third_party_settings: {  }
+  field_di_categories:
+    type: entity_reference_label
```

Comment:
Does this still need changing back? See https://github.com/dof-dss/nidirect-drupal/pull/249#discussion_r392134139

Reply:
If you want it consistent with the full display, yes this would need to be changed.

Reply:
The html_list_formatter is outputting the list of categories without a label. This is perfect for the teaser display (the full display isn't really being used as driving instructor nodes are only viewed in driving instructor search results using the teaser display) - but it looks incongruous with the output of the other fields in the diff display which are output with a label and output is controlled with field.html.twig and diff display specific field--node--diff.html.twig templates.